Library/Homebrew/cmd/list.rb

```diff
@@ -76,7 +76,8 @@ def list
     lib/ruby/gems/[12].*
     lib/ruby/site_ruby/[12].*
     lib/ruby/vendor_ruby/[12].*
-    manpages/brew.1
+    manpages/man1/brew.1
+    manpages/man1/brew-cask.1
```

Comment:
Do we have any non-`.1` cases currently, though? If not, I'd rather leave as-is (or at least make that another PR).